dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -815,32 +828,32 @@
           "type": "string",
           "description": "Healthcare Entity Category.",
           "enum": [
-            "BODY_STRUCTURE",
```

Comment:
These changes must get into 2022-05-01 and, if you're in it, 2022-05-15-preview (think that's just CLU, though).